server/controllers.js

```diff
@@ -56,3 +56,26 @@ module.exports.put = (endpoint, config) => {
     });
 };
 
+module.exports.post = (endpoint, config) => {
+
+  var { body, route, type } = config;
+
+  //FOR REVIEWS
+  // var url = new URL(join(api, route[0], param[0], route[1])).href;
+
+  //FOR QUESTIONS
+  //(type 'q')
+  var url = new URL(join(api, route[0])).href;
+  if (type === 'a') { // 'qa/questions',    ID #    ,  'answers'
+    url = new URL(join(api, route[0], body.product_id, route[1])).href;
+  }
+
+  const options = { headers: { Authorization: process.env.GITHUB_API_KEY } };
+  console.log(body);
+  // return a promise for the requested object
+  return axios.post(url, body, options)
+    .then((res) => res.body)
+    .catch((err) => {
+      console.log(err.toJSON());
```

Comment:
use `console.error` to log errors

Reply:
this logs the actual error vs using console error new Error. it implements the built in function of express

Reply:
console.error highlights the log in red on the console, the change I am suggesting is to use:
```javascript
console.error(err.toJSON());
```